react/features/base/ui/components/web/MultiSelect.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useCallback, useMemo, useRef } from 'react';
+import { makeStyles } from 'tss-react/mui';
+
+import { IconCloseLarge } from '../../../icons/svg';
+import Popover from '../../../popover/components/Popover.web';
+import { MultiSelectItem } from '../types';
+
+import ClickableIcon from './ClickableIcon';
+import ContextMenu from './ContextMenu';
+import ContextMenuItem from './ContextMenuItem';
+import ContextMenuItemGroup from './ContextMenuItemGroup';
+import Input from './Input';
+
+type Props = {
+    autoFocus?: boolean;
+    disabled?: boolean;
+    filterValue?: string;
+    hideList: Function;
+    isOpen?: boolean;
+    items: MultiSelectItem[];
+    noMatchesText?: string;
+    onFilterChange?: (value: string) => void;
+    onRemoved: (item: any) => void;
+    onSelected: (item: any) => void;
+    placeholder?: string;
+    selectedItems?: MultiSelectItem[];
+};
+
+const useStyles = makeStyles()(theme => {
+    return {
+        contextMenu: {
+            position: 'relative' as const,
+            marginTop: '8px',
+            right: 'auto',
+            padding: '0',
+            boxShadow: '0px 5px 10px rgba(0, 0, 0, 0.75)'
+        },
+        selectedList: {
+            position: 'static' as const,
+            padding: 0,
+            maxHeight: '200px',
+            marginTop: '8px',
+            boxSizing: 'border-box'
+        },
+        selectedItem: {
+            display: 'flex',
+            justifyContent: 'space-between',
+            width: '100%',
+            '&:hover': {
+                backgroundColor: theme.palette.ui01
+            },
+            cursor: 'auto',
+            paddingTop: '0px',
+            paddingBottom: '0px',
+            paddingLeft: '16px',
+            paddingRight: '6px'
+        },
+        contentListItem: {
+            inlineSize: 'calc(100% - 38px)',
+            overflowWrap: 'break-word'
+        },
+        contentSelectedItem: {
+            inlineSize: 'calc(100% - 82px)',
```

Comment:
Same question as above

Reply:
the size of the clear icon and the spacing

Reply:
the size of the icon + the clear icon and the spacing